src/webpage/.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+target = "wasm32-unknown-unknown"
```

Comment:
will this break the desktop mode ?

Reply:
won't break, but it will make vscode consider the target "wasm32-unknown-unknown" when running it's LSP.